tox.ini

```diff
@@ -1,27 +1,30 @@
 [tox]
-envlist = 
-    py27,py33,py34,py35,pypy,pypy3,coverage,docs
+envlist =
+    py27,py34,py35,py36,pypy,pypy3,coverage,docs
 
 [testenv]
-commands = 
+commands =
     python setup.py -q test -q
+    sphinx-build -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+deps =
+    .[test,docs]
 
 [testenv:coverage]
 usedevelop = true
 basepython =
     python2.7
-commands = 
-    nosetests --with-xunit --with-xcoverage
+commands =
+    coverage run setup.py -q test -q
+    coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+    coverage report
```

Comment:
TravisCI doesn't run 'coverage report' because in my opinion coveralls does a better job reporting *and* failing the build. It's nice to be able to see that one red X for coveralls rather than having to go look at the output of each individual Travis build to see why it failed. (In other words, its good not to conflate test failures with coverage failures.)

I suppose we could add that to tox.ini going forward. It doesn't make much of a difference to me; I tend to *look* at the coverage report (`coverage html`) before pushing anything substantial---I have to do that anyway if coverage drops---but if I forget and coverage drops, coveralls catches it 😄 

The primary downside I see is those cases where complete coverage requires several combined tox environment runs (different python versions). I know we have at least two projects that handle that case, but when I tried one of them it didn't work very well for me, at least not using `detox` (which is mainly how I run tox) (something was messed up in the report, I don't exactly remember what).